modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AssetManagerWithSecurity.java

```diff
@@ -90,7 +91,12 @@ private boolean isAuthorizedByAcl(Version version, String mpId, String action) {
     Opt<Asset> secAsset = super.getAsset(version, mpId, "security-policy-episode");
     if (secAsset.isSome()) {
       InputStream in = secAsset.get().getInputStream();
-      final AccessControlList acl = authSvc.getAclFromInputStream(in).getA();
+      final AccessControlList acl;
+      try {
+        acl = authSvc.getAclFromInputStream(in);
+      } catch (IOException e) {
+        return false;
```

Comment:
Hmm... there seem to be several problems that could cause this exception:

- Access to "in" works but data is corrupt ("cannot be parsed")
- Access to "in" does not work, e.g. unauthorised access

Maybe handle those cases separately and log an error in case the data is corrupt?

Reply:
Logging makes sense. Done